### PR TITLE
Translate 1 file to zh - tsconfig.json

### PR DESCRIPTION
**Pull request by originalix**
Files:
- tsconfig.json.md

### CODE REVIEW

**Comment by Kingwl** [comment]
Thanks for the contribution! I'll take a look soon.

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      "sourceMap": true
+    },
+    "include": ["src/**/*"],
+    "exclude": ["node_modules", "**/*.spec.ts"]
+  }
+  ```
+
+## TSConfig 基础
+
+根据你想要执行代码的 JavaScript 运行时环境，可能有一个基本配置，你可以参考[github.com/tsconfig/bases](https://github.com/tsconfig/bases/)来使用。
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:77]: 根据你要在其中运行代码的不同的 JavaScript 运行时环境，你可以在 [github.com/tsconfig/bases]() 上寻找一个合适的基本配置。你可以通过扩展这些已经处理过不同 JavaScript 运行时环境的 `tsconfig.json` 文件来简化你项目中的 tsconfig.json 。

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+这些你的项目扩展的 `tsconfig.json` 文件，它通过运行时的支持来简化你的 `tsconfig.json` 文件配置。
+
+<!-- 这些是你的项目扩展的""文件，它通过处理运行时支持简化你的“” -->
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:80]: Should be removed?

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+举个例子，如果你的项目是基于 Node.js 12.x 写的，那么你可以使用 npm 依赖：[`@tsconfig/node12`](https://www.npmjs.com/package/@tsconfig/node12):
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:81]: npm 模块

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  "compilerOptions": {
+    "preserveConstEnums": true
+  },
+
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "**/*.spec.ts"]
+}
+```
+
+这使你的 `tsconfig.json` 专注在你自己选择的项目环境上，而不是所有的运行时环境。现在已经有了一些 tsconfig 基础配置，我们希望社区能够为不同的环境添加更多的内容。
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:96]: 专注在你的项目的目标环境上

Comment (Kingwl) [reply docs/documentation/zh/project-config/tsconfig.json.md:96]: 而不是所有可能的运行时环境

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+- [Node 10](https://www.npmjs.com/package/@tsconfig/node10)
+- [Node 12](https://www.npmjs.com/package/@tsconfig/node12)
+- [Node 14](https://www.npmjs.com/package/@tsconfig/node14)
+- [Deno](https://www.npmjs.com/package/@tsconfig/deno)
+- [React Native](https://www.npmjs.com/package/@tsconfig/react-native)
+- [Svelte](https://www.npmjs.com/package/@tsconfig/svelte)
+
+## 细节
+
+当`"compilerOptions"` 属性忽略的时候，会使用编译器的默认配置。请参考我们支持的[编译器选项](/tsconfig)列表
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:108]: 的时候 -> 时

Comment (Kingwl) [reply docs/documentation/zh/project-config/tsconfig.json.md:108]: 列表`。`

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+## TSConfig 参考
+
+想要了解更多的配置选项的信息，请访问 [TSConfig Reference](/tsconfig)
+
+## 模式
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:114]: 协议?

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+`tsconfig.json` 的模式可以在这里找到 [the JSON Schema Store](http://json.schemastore.org/tsconfig).
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:116]: 协议？

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+- 调用 tsc 命令并且没有其它输入文件参数时，在这种情况下编译器会搜索 `tsconfig.json` 文件，从当前目录开始搜索，一直搜寻到父级目录。
+
+- 调用 tsc 命令并且没有其他输入文件参数，使用 `--project` （或者只是 `-p`）的命令行选项来指定包含了 `tsconfig.json` 的目录，或者包含有效配置的 `.json` 文件路径。
+
+当命令行中指定了输入文件参数， `tsconfig.json` 文件会被忽略。
+
+## 示例
+
+`tsconfig.json` 文件示例：
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:27]: ：

File: docs/documentation/zh/project-config/tsconfig.json.md (hunk 3 above)
Comment (Kingwl) [reply docs/documentation/zh/project-config/tsconfig.json.md:81]: ：

File: docs/documentation/zh/project-config/tsconfig.json.md (hunk 6 above)
Comment (Kingwl) [reply docs/documentation/zh/project-config/tsconfig.json.md:114]: 请访问[]() `。`

File: docs/documentation/zh/project-config/tsconfig.json.md (hunk 7 above)
Comment (Kingwl) [reply docs/documentation/zh/project-config/tsconfig.json.md:116]: `.` -> `。`

**Review by Kingwl — CHANGES_REQUESTED** [review]

**Comment by Kingwl** [comment]
@laosb PTAL.

**Comment by originalix** [comment]
@Kingwl 😄 Thanks for code review. I've corrected all the questions.

File: docs/documentation/zh/project-config/tsconfig.json.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+- [Node 10](https://www.npmjs.com/package/@tsconfig/node10)
+- [Node 12](https://www.npmjs.com/package/@tsconfig/node12)
+- [Node 14](https://www.npmjs.com/package/@tsconfig/node14)
+- [Deno](https://www.npmjs.com/package/@tsconfig/deno)
+- [React Native](https://www.npmjs.com/package/@tsconfig/react-native)
+- [Svelte](https://www.npmjs.com/package/@tsconfig/svelte)
+
+## 细节
+
+当`"compilerOptions"` 属性忽略时，会使用编译器的默认配置。请参考我们支持的[编译器选项](/tsconfig)列表。
```
Comment (Kingwl) [inline docs/documentation/zh/project-config/tsconfig.json.md:107]: > 属性忽略时

seems `当没有指定 compilerOptions 时` is better.

Comment (originalix) [reply docs/documentation/zh/project-config/tsconfig.json.md]: Done!

**Comment by Kingwl** [comment]
Looks good. But I think we may need one more eye to review again.

**Comment by Kingwl** [comment]
Seems there's no more eyes.
It's looks basically good. I think we could move on.

So

LGTM.

**Comment by orta** [comment]
I can always ask for more Chinese readers on twitter?

**Comment by Kingwl** [comment]
I'm planning seeking for help at some Chinese social network too.

**Pull request merged**